_sips/minutes/2020-01-27-sip-minutes.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: sips
+title: SIP Meeting Minutes - January 27 2019
+
+partof: minutes
+---
+
+# Minutes
+
+The meeting took place without a pre-defined agenda.
+
+## Date and Location
+
+The meeting took place on the 27th January 2020 at 17:00 CET via Zoom at EPFL in Lausanne, Switzerland, as well
+as other locations.
+
+The meeting was broadcast and recorded on the Scala Process's YouTube channel:
+[SIP meeting January 2020](https://www.youtube.com/watch?v=ws2AaDUg-6E) [1:00:53].
+
+Minutes were taken by Dale Wijnand.
+
+## Attendees
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), Scala Center
+* Guillaume Martres ([@smarter](https://github.com/smarter), EPFL
+* Adriaan Moors ([@adriaanm](https://github.com/adriaanm)), Lightbend
+* Seth Tisue ([@SethTisue](https://github.com/SethTisue)), Lightbend
+* Josh Suereth ([@jsuereth](https://github.com/jsuereth)), Independent
+* Dale Wijnand ([@dwijnand](https://twitter.com/dwijnand)), secretary
+
+## Not present
+
+* Darja Jovanovic ([@darjutak](https://github.com/darjutak)), Process Lead
+* Iulian Dragos ([@dragos](https://github.com/dragos)), Triplequote
+* Lukas Rytz ([@lrytz](https://twitter.com/lrytz)), Lightbend
+* Miles Sabin ([@milessabin](https://github.com/milessabin)), Independent
+* Heather Miller ([@heathermiller](https://github.com/heathermiller)), CMU
+
+## Proceedings
+
+(Unforunately, while the lovely external microphone was fully functional for the Zoom call, unknowing to
+everyone at the meeting (until near the end), the laptop microphone was what was being used for the YouTube OBS
+capture.  Therefore, it is near impossible to hear what was said from the YouTube recording, so the minutes this
+month will be an opinionated executive summary of only the hightlights.)
+
+### Binary Integer Literal
+
+Thread: <https://contributors.scala-lang.org/t/pre-sip-support-binary-integer-literals/3559>
+
+The committee feels that, despite the benefits outlined in the proposal in this Pre-SIP, it doesn't fit the
+criteria for inclusion in Scala 3.0, specifically given it can be added later without technical breakages or
```

Comment:
> criteria inclusion in Scala 3.0

there is an ambiguity here whether this means 3.0 specifically, or any Scala 3 release. saying "3.0" is probably not enough to dispel this ambiguity in the minds of all readers